app/web/src/components/IdeEditor/IdeEditor.tsx

```diff
@@ -17,6 +18,7 @@ const IdeEditor = ({ Loading }) => {
     cadquery: 'python',
     openscad: 'cpp',
     jscad: 'javascript',
+    INIT: '',
```

Comment:
I wasn't sure how to handle the INIT type but I didn't want those red squiggles anymore.